src/nidcpower/metadata/attributes_addon.py

```diff
@@ -2,5 +2,39 @@
 # Any changes to the API should be made here. attributes.py is code generated
 
 attributes_override_metadata = {
+    1150208: {"codegen_method": "no"},
```

Comment:
please add TODO pointing to Issue for adding these

Reply:
I've filed an issue for this and added a TODO comment referencing that issue in the latest commit.